SU-SWS/stanford_profile
# D8CORE-369: Enable metatags and XMLSitemap on prod

# READY FOR REVIEW/ 

# Summary
- Enables xmlsitemap, metatag, and config_ignore
- xmlsitemap creates config items for each sitemap it creates. I went with the config ignore approach please comment on that. 

# Review By (Date)
- Feb 12th

# Urgency
- Med/High as it is needed for RC1

# Steps to Test

1. Check out this branch and D8CORE-369 branch of acsf-cardinalsites
2. Do an install
3. Validate the configuration

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- D8CORE

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
